pkg/controller/kieapp/constants/constants.go

```diff
@@ -73,12 +73,18 @@ const (
 	LabelRHsubcomponentType = "rht.subcomp_t"
 	// LabelRHcompany used as metering label
 	LabelRHcompany = "com.company"
+	// IBMBamoeImageContext IBM image context
+	IBMBamoeImageContext = "ibm-bamoe"
+	// IBMBamoeImagePrefix IBM image prefix
+	IBMBamoeImagePrefix = "bamoe"
 	// RhpamPrefix RHPAM prefix
 	RhpamPrefix = "rhpam"
 	// RhdmPrefix RHDM prefix
 	RhdmPrefix = "rhdm"
```

Comment:
Is still needed ?

Reply:
yes, due the environments, rhdm for decisions and rhpam for process.
better not to rename it at this time.